### PR TITLE
add note that blog bundle is abandoned

### PR DESCRIPTION
**Pull request by dbu**
* [ ] mark https://packagist.org/packages/symfony-cmf/blog-bundle as abandoned
* [x] alter the github project description to say this is abandoned
* [x] check symfony-cmf-docs and remove all mentions of the BlogBundle https://github.com/symfony-cmf/symfony-cmf-docs/pull/757

### CODE REVIEW

**Comment by dbu** [comment]
question: should we delete the other content of the readme or should we leave that in?

File: README.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
 # Symfony CMF Blog Bundle
 
-[![Build Status](https://travis-ci.org/symfony-cmf/BlogBundle.png?branch=master)](https://travis-ci.org/symfony-cmf/BlogBundle)
-[![Latest Stable Version](https://poser.pugx.org/symfony-cmf/blog-bundle/v/stable.png)](https://packagist.org/packages/symfony-cmf/blog-bundle)
-[![Total Downloads](https://poser.pugx.org/symfony-cmf/blog-bundle/downloads.png)](https://packagist.org/packages/symfony-cmf/blog-bundle)
+ > **WARNING: Unmaintained** The Blog Bundle was built to demonstrate CMF features. To focus our efforts
+ > in the CMF project, we decided to abandon this bundle, it is not maintained anymore.
```
Comment (WouterJ) [inline README.md]: "[...] abandon this bundle. It is no longer maintained."

**Comment by WouterJ** [comment]
> question: should we delete the other content of the readme or should we leave that in?

I think we should leave it there. This way, we can always start working on this again.

Btw, for the other abandoned bundles, we have to use a slightly other wording as we will still release bug fixes (so it's still somewhat maintained, but no new features should be expected any time soon).

**Comment by dbu** [comment]
improved the text as suggested.

agree that bundles that have releases will need a more precise text.

**Comment by WouterJ** [comment]
:+1:

**Pull request merged**